src/models/formFields.ts

```diff
@@ -0,0 +1,17 @@
+export interface FormFields {
+  username: string;
+  name: string;
+  email: string;
+  age: number;
+  skills: any;
+  commitment: number;
+  additionalInfo: string;
+  resumeUrl: string;
+  experience: string;
+  reason: string;
+  accepted: boolean;
+  zip: number;
+  role: string;
+  id: string;
+  project: string;
```

Comment:
this is an optional field

Reply:
If you are referring to project then its not a field in the form at all but does have a set value from a slug. So im not sure about this comment.